engines/griffon/dialogs.cpp

```diff
@@ -256,7 +256,7 @@ void GriffonEngine::configMenu() {
 
 	int ticks1 = _ticks;
 
-	_cloudImg->setAlpha(128, true);
+	// _cloudImg->setAlpha(128, true);
```

Comment:
Since you're replicating the behaviour it's better to remove these lines than comment out.